qemu/tests/hv_enforce_cpuid.py

```diff
@@ -0,0 +1,57 @@
+
+from virttest import error_context
+from provider import qemu_img_utils
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Add 'hv-enforce-cpuid' option to try to use a forbidden feature.
+    1. Boot the guest with following hyper v flags.
+    2. Run command rdmsr 0x40000002 and result should be like following: CPU 0 cannot read MSR 0x40000002.
+    3. Boot the guest without hyper v flags.
+    4. Run command rdmsr 0x40000002 and result should be 0.
+    5. Check the results
+
+    param test: the test object
+    param params: the test params
+    param env: the test env object
+    """
+
+    def _get_output_from_vm(test, params, env):
+        """
+        Boot the guest with cpu_model_flags,
+        then return cmd output.
+
+        param test: the test object
+        param params: the test params
+        param env: the test env object
+        return: cmd output
+        """
+        vm = qemu_img_utils.boot_vm_with_images(test, params, env)
+        session = vm.wait_for_login(timeout=360)
+        # modify the subscription-manager
+        session.cmd_status_output(params.get("modify_submanager_cmd"))
+        # install repo
+        session.cmd_status_output(params.get("repo_install_cmd"))
+        # install msr-tools
+        status = session.cmd_status_output(params.get("msr_install_cmd"))
```

Comment:
if we don't want to check the output of a command, we can also use session.cmd_status(cmd), e.g. 
if session.cmd_status(cmd) != 0:
    # cmd failed